samples/apps/agent-id-sample/README.md

```
@@ -172,14 +188,26 @@ What flights are there from Colombo to Singapore?
 
 These browsing tools use the agent's M2M token — no popup beyond the initial sign-in.
 
+You can also ask the agent for general recommendations — for example:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)